crates/client/gateway/server/src/service.rs

```diff
@@ -36,46 +36,32 @@ pub async fn start_server(
 
     tracing::info!("🌐 Gateway endpoint started at {}", addr);
 
-    let shutdown_notify = Arc::new(Notify::new());
-
-    {
-        let shutdown_notify = Arc::clone(&shutdown_notify);
-        tokio::spawn(async move {
-            graceful_shutdown(&ctx).await;
-            shutdown_notify.notify_waiters();
-        });
-    }
-
     loop {
```

Comment:
(not related) huh i thought all of this was using hyper, why are we listener.accept() stuff by hand here?

Reply:
Linking @jbcaron here.

Reply:
I think we don’t have a choice with the new version of Hyper.